pkg/kopia/command/kopia.go

```diff
@@ -59,3 +71,188 @@ func commonArgs(password, configFilePath, logDirectory string, requireInfoLevel
 func ExecKopiaArgs(configFilePath string) []string {
 	return commonArgs("", configFilePath, "", false).StringSliceCMD()
 }
+
+func ResolveS3Endpoint(endpoint string) string {
+	if strings.HasSuffix(endpoint, "/") {
+		log.Debug().Print("Removing trailing slashes from the endpoint")
+		endpoint = strings.TrimRight(endpoint, "/")
+	}
+	sp := strings.SplitN(endpoint, "://", 2)
+	if len(sp) > 1 {
+		log.Debug().Print("Removing leading protocol from the endpoint")
```

Comment:
Add more info to the logs to differentiate if there are multiple instances of logs